### PR TITLE
runtime/predicates: Add unit tests

### PR DESCRIPTION
**Pull request by darkowlzz**
Adds unit tests for ReconcileRequestedPredicate.Update.

### CODE REVIEW

File: runtime/predicates/reconcile_requested_test.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+package predicates
```
Comment (hiddeco) [inline runtime/predicates/reconcile_requested_test.go:1]: Can you please add a copyright header? We do this for any Go file in our projects.

Comment (darkowlzz) [reply runtime/predicates/reconcile_requested_test.go:17]: Added.

**Review by hiddeco — APPROVED** [review]
:bowing_man::clap: x :100:

**Comment by hiddeco** [comment]
(Rebase again, I am back from vacation so this should happen less often in the future).

**Comment by darkowlzz** [comment]
np, rebased :slightly_smiling_face: Thanks

**Pull request merged**